common/scala/src/main/scala/org/apache/openwhisk/core/service/LeaseKeepAliveService.scala

```diff
@@ -0,0 +1,16 @@
+package org.apache.openwhisk.core.service
+
+// Data
+sealed trait KeepAliveServiceData
+case object NoData extends KeepAliveServiceData
+case class Lease(id: Long, ttl: Long) extends KeepAliveServiceData
+
+// Events received by the actor
+case object ReGrantLease
+case object GetLease
+case object GrantLease
+
+// TBD
+class LeaseKeepAliveService {
```

Comment:
I realized this pr is also dependent on these modules.
It would be better to merge this PR after this kind of module is introduced.